lib/widgets/scrollable_genres.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+
+import '../utils/constants.dart';
+
+class ScrollableGenres extends StatelessWidget {
+  const ScrollableGenres({
+    super.key,
+    required this.genres,
+  });
+
+  final List<String> genres;
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      scrollDirection: Axis.horizontal,
+      child: Center(
+        child: Expanded(
```

Comment:
Question, why those two widgets are needed? I can't figure out 😞

Reply:
They aren't. I tried a few different approaches when i began implementing the scrollable genres and now i noticed that the dont really change anything in my code. Maybe in the last module i could use one of them if i implement any of the ideas i told you about. Fixed.